src/features/metadata-based-view/MetadataBasedItemList.js

```diff
@@ -171,29 +190,61 @@ class MetadataBasedItemList extends React.Component<Props, State> {
         );
     };
 
+    scrollPositionClasses(width: number): ScrollPositionClasses {
```

Comment:
Since these are functions, should they start with verbs, e.g., `getScrollPositionClasses` and `calculateTotalContentWidth`?